core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/deployment/CloudflowApplication.scala

```diff
@@ -103,8 +100,7 @@ object CloudflowApplication {
   )
 
   implicit val statusSubEnabled = CustomResource.statusMethodsEnabler[CR]
-
-  val CRD = CustomResourceDefinition[CR]
+  val CRD                       = CustomResourceDefinition[CR]
```

Comment:
We should move the CRD and CR skuber definitions back to operator, (and leave the formats in blueprint)

Reply:
Moved back `CloudflowApplication` to `operator` since we now use `ApplicationDescriptor` for CR generation .. hence done ..